anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/DetectorManagerTest.java

```diff
@@ -145,14 +151,8 @@ private void initTestObjects() {
     private void initDependencies() {
         when(detector.classify(goodMetricData)).thenReturn(anomalyResult);
 
-        when(detectorSource.findDetectorTypes())
-                .thenReturn(Collections.singleton(DETECTOR_TYPE));
-
-        when(detectorSource.findDetector(any(UUID.class), eq(goodDefinition)))
-                .thenReturn(detector);
-        when(detectorSource.findDetector(any(UUID.class), eq(badDefinition)))
```

Comment:
Any reason not to test this?  This has caused coverage of DetectorManager coverage to drop by 8%:
https://codecov.io/gh/ExpediaDotCom/adaptive-alerting/pull/452/src/anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/DetectorManager.java#L120

Reply:
Thanks, this was a miss.